tap_twilio/schema.py

```diff
@@ -31,6 +33,11 @@ def get_schemas():
             valid_replication_keys=stream_metadata.get('replication_keys', None),
             replication_method=stream_metadata.get('replication_method', None)
         )
+
+        # If this is not added, the streams are skipped when loading for the first time
+        # https://github.com/singer-io/tap-shopify/issues/40
+        # However this is just a hack and I might not be understanding something. TODO correctly address this
+        mdata[0]['metadata']['selected'] = True
```

Comment:
This is the expected behaviour, so please don't mark the the streams to be selected here. The main PipelineWise component will do it programmatically and will update the schema JSONs automatically using the list of tables (aka streams - example [here](https://github.com/transferwise/analytics-platform-config/blob/d96b1352c2e7c25191677e83352f85b82505b607/pipelinewise/tap_zendesk.yml#L37)) that are defined in the [tap YML files](https://github.com/transferwise/analytics-platform-config/tree/master/pipelinewise).

By default no streams are selected and that is fine that way. When developing locally you can add this `mdata[0]['metadata']['selected'] = True` manually to the generated (`--discover`) JSON files.